pkg/ccl/importccl/BUILD.bazel

```diff
@@ -107,6 +107,7 @@ go_test(
     ],
     data = glob(["testdata/**"]),
     embed = [":importccl"],
+    shard_count = 16,
```

Comment:
If we want to pin this diff (and we do), we can tell bazel to pin the diff.
```
shard_count = 16,  # keep
```

Mind just adding a tiny comment right above here explaining what this does? Perhaps even directly linking to https://docs.bazel.build/versions/master/be/common-definitions.html#test.shard_count. We'll probably need to update most of our test packages to do the same.

Reply:
i think gazelle already pins some fields, it's just deps that gets regenerated.

Reply:
Ah, that makes sense.